src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java

```diff
@@ -958,8 +959,7 @@ public void onWomGroupSynced(WomGroupSynced event)
 	@Subscribe
 	public void onWomGroupMemberAdded(WomGroupMemberAdded event)
 	{
-		MemberInfo member = event.getMember();
-		groupMembers.put(member.getUsername().toLowerCase(), member);
+		womClient.importGroupMembers();
```

Comment:
Is this really required?

Reply:
This has to be done for the same reason as the one above.